inputs.css

```diff
@@ -16,7 +16,7 @@
     position: absolute;
     font-size: inherit;
     fill: var(--gray-2);
-    left: 0.5em;
+    left: 2.5em;
```

Comment:
Why change this here? This PR shouldn't be worries about any inputs, right?

Reply:
I checked the main branch and it's a bug. I fixed it in this commit.